assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx

```diff
@@ -113,8 +113,35 @@ const content = (
         </ProductPageTextBlock>
         <WeeklyForm />
         <ProductPageInfoChip>
-          <p>Gifting is available for quarterly and annual subscriptions</p>
-          <p>You can cancel your subscription at any time</p>
+          <span className="component-product-page-info-chip__gifting-info">
```

Comment:
Hi, I'm sorry to appear this late, but `ProductPageInfoChip ` already takes an `icon` parameter that renders an inline SVG in the right place, in a more reusable way